.env

```diff
@@ -1 +1 @@
-VUE_APP_API_ROOT=http://localhost:8080/api/v1
+VUE_APP_API_ROOT=http://localhost:8095/api/v1
```

Comment:
What's the reason for this change?

Reply:
I just didn't go to the trouble of creating `.env.local` like a reasonable person and it got swept in.